source/type/vehicle_journey.h

```diff
@@ -107,6 +107,7 @@ struct VehicleJourney : public Header, Nameable, hasVehicleProperties {
 
     // Return the smallest time within its stop_times
     uint32_t earliest_time() const;
+    uint32_t last_time() const;
```

Comment:
To match earliest ('last' matches 'first'):
```suggestion
    uint32_t latest_time() const;
```

Reply:
I think here last doesn't mean latest but destination arrival_time -> date_time of end station.

Reply:
Then it's the same for "earliest" and it should be "first"? Either is fine, but it looks like it should match.